.circleci/config/jobs/pre-flight-checks.yml

```diff
@@ -24,6 +24,13 @@ steps:
         . $BASH_ENV
         which circleci
         circleci version
-  - run: make ci-verify
+  - run:
+      name: Verify CircleCI
+      command: |
+        export CCI_PATH=/tmp/circleci-cli/$CIRCLECI_CLI_VERSION
+        echo "export PATH=$CCI_PATH:\$PATH" >> $BASH_ENV
+        set -x
+        . $BASH_ENV
+        make ci-verify
```

Comment:
Setting the path seems necessary as the job was failing with 
`Please install CircleCI CLI. See https://circleci.com/docs/2.0/local-cli/#installation `

Reply:
The previous step updates BASH_ENV.